openapi/openapi.yaml

```diff
@@ -24,6 +24,10 @@ tags:
     description: Internal account management endpoints for creating and managing internal accounts.
   - name: External Accounts
     description: External account management endpoints for creating and managing external bank accounts
+  - name: Plaid
```

Comment:
nit: I might consider just including this in the external accounts tag instead of adding another tag group. It's just getting a bit crowded over there in the left nav